src/services/Warnings.js

```diff
@@ -87,11 +87,16 @@ const getWarningsZones = (warningsZone, zone, client) => {
     const strBeginDate = beginTime.format('DDMMMYY').toUpperCase();
     const strEndDate = endTime.format('DDMMMYY').toUpperCase();
 
-    const strTime = beginTime.isSame(endTime, 'day')
-      ? `${strBeginHour} e as ${endTime.calendar(actualTime)} ${strBeginDate}`
-      : `${beginTime.calendar(actualTime)} ${strBeginDate} e as ${endTime.calendar(actualTime)} ${strEndDate}`;
+    const strTime = (() => {
+      if (beginTime.isAfter(actualTime)) {
+        return `até às ${endTime.calendar(actualTime)} ${strEndDate}`;
+      } if (beginTime.isSame(endTime, 'day')) {
```

Comment:
Put this in a new line.

```suggestion
      }
      
      if (beginTime.isSame(endTime, 'day')) {
```